matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/call/model/MxCallImpl.kt

```diff
@@ -205,8 +205,8 @@ internal class MxCallImpl(
         Timber.tag(loggerTag.value).v("select answer $callId")
         if (!isOutgoing) return
         // This is an outgoing call, select the remote client that answered.
-        if (state != CallState.Dialing) {
-            Timber.tag(loggerTag.value).w("Expected state is CallState.Dialing got $state.")
+        if (state !is CallState.Dialing && state !is CallState.Connected) {
```

Comment:
`!= Dialing` is still valid here as `Dialing` is an `object`, or was this done for consistency? I lean towards using `!=` where possible as it provides extra context, but not a strong opinion

Reply:
Yea no strong opinion here either, happy to change.